### PR TITLE
Update order of exercises in CMakeLists.txt

### PR DESCRIPTION
**Pull request by rahul007-bit**
Please review the changes, these changes are related to this #3 issue.

### CODE REVIEW

File: exercises/02_functions/CMakeLists.txt [hunk 1]
```diff
@@ -1,21 +1,22 @@
-##############################################
+# #############################################
```
Comment (rdjondo) [inline exercises/02_functions/CMakeLists.txt]: It looks like there are new space added in the comments. This is done at many places. Could you revert those please?

**Review by rdjondo — CHANGES_REQUESTED** [review]
Could revert the spaces added after the "#" characters? They are not necessary and make the pull request bigger.

File: exercises/02_functions/CMakeLists.txt (hunk 1 above)
Comment (rahul007-bit) [reply exercises/02_functions/CMakeLists.txt]: I have reverted the unintentional whitespace additions in the comments.

**Comment by rdjondo** [comment]
Thanks @rahul007-bit

**Pull request merged**